functests/performance/performance.go

```diff
@@ -92,13 +92,13 @@ var _ = Describe("performance", func() {
 
 		// Check /usr/local/bin/pre-boot-tuning.sh existence under worker's rootfs
 		const perfRtKernelPrebootTuningScript = "/usr/local/bin/pre-boot-tuning.sh"
```

Comment:
don't really understand why we need this const, but.. probably it's out of scope of this PR